examples/admissioncontrol_policy/README.md

```diff
@@ -0,0 +1,66 @@
+```go
+package main
+
+import (
+	"fmt"
+	"github.com/dome9/dome9-sdk-go/dome9"
+	"github.com/dome9/dome9-sdk-go/services/admissioncontrol/admission_policy"
+)
+
+func main() {
+	// Pass accessID, secretKey, rawUrl, or set environment variables
+	config, _ := dome9.NewConfig("Access-ID", "Secret-Key", "https://api.dome9.com/v2/")
+	srv := admission_policy.New(config)
+
+	admissionControlCreateRequest := admission_policy.AdmissionControlPolicyRequest{
+		TargetId:        "<K8S_Cluster_ID>",
+		TargetType:      "Environment",
+		RulesetId:       "<RuleSetID>",
+		NotificationIds: []string{"<NOTIFICATION_IDS>"},
+		Action:          "Prevention",
+		RulesetPlatform: "kubernetesruntimeassurance",
+	}
+
+	// CreateAdmission Control Policy
+	admissionControlPolicyResponse, _, err := srv.Create(&admissionControlCreateRequest)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Printf("Create AC Policy response type: %T\n Content: %+v", admissionControlPolicyResponse, admissionControlPolicyResponse)
+
+	// Get all Admission Control Policy Associations
+	allAdmissionControlPolicies, _, err := srv.GetAllAdmissionControlPolicies()
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Printf("GetAll Admission Control Policies response type: %T\n Content: %+v", allAdmissionControlPolicies, allAdmissionControlPolicies)
+
+	// Get specific Admission Control Policy
+	admissionControlPolicy, _, err := srv.GetAdmissionControlPolicy(admissionControlPolicyResponse.ID)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Printf("Get Specific Admission Control Policy response type: %T\n Content: %+v", admissionControlPolicy, admissionControlPolicy)
+
+	// Update specific Admission Control Policy
+	admissionControlCreateRequest.Action = "Detection"
+	admissionControlUpdatedPolicy, _, err := srv.UpdateAdmissionControlPolicy(&admissionControlCreateRequest)
+	if err != nil {
+		fmt.Println(err)
+	}
+	fmt.Printf("Update AC Policy response type: %T\n Content: %+v", admissionControlUpdatedPolicy, admissionControlUpdatedPolicy)
+
+	// Delete AC Policy
+	deleteResponse, err := srv.Delete(admissionControlPolicyResponse.ID)
+	if err != nil {
+		panic(err)
```

Comment:
any idea in which error we should just print and in which we should panic?

Reply:
here the type of the error really doen't matter since it is only an example, i can change it to print only instead of panic

Reply:
I don't really care. I just wondered when each is used

Reply:
i think no specific cases needed, but it is better to stop execution when one of the tests inside the example is failed, so i would like to change all of them to panic to stop execution flow